ccdproc/tests/test_combiner.py

```diff
@@ -393,3 +393,23 @@ def test_writeable_after_combine(ccd_data, tmpdir, comb_func):
     ccd2 = getattr(combined, comb_func)()
     # This should not fail because the resulting uncertainty has a mask
     ccd2.write(tmp_file.strpath)
+
+def test_iraf_minmax_masking():
+    ccdlist = [CCDData(np.ones((3, 5))*90., unit="adu"),\
+               CCDData(np.ones((3, 5))*20., unit="adu"),\
+               CCDData(np.ones((3, 5))*10., unit="adu"),\
+               CCDData(np.ones((3, 5))*40., unit="adu"),\
+               CCDData(np.ones((3, 5))*25., unit="adu"),\
+               CCDData(np.ones((3, 5))*35., unit="adu"),\
+              ]
+    ccdlist[0].data[0,1] = 3.1
+    ccdlist[1].data[1,2] = 100.1
+    ccdlist[1].data[2,0] = 100.1
+    c = Combiner(ccdlist)
+    c.iraf_minmax_clipping(nlow=1, nhigh=1)
+    result = c.average_combine()
+    expected = [[ 30.,  22.5, 30. , 30., 30.],\
+                [ 30. , 30. , 47.5, 30., 30.],\
+                [ 47.5, 30. , 30. , 30., 30.]]
+    expected = np.array(expected)
+    assert (result == expected).all()
```

Comment:
There is some ``numpy`` testing utility for this ``np.testing.assert_array_equal`` (or ``np.testing.assert_all_close`` if there might be floating point inequality). Then you don't need to wrap ``expected`` as ``numpy.array``:

```
expected = [[30, ...], ...]
np.testing.assert_array_equal(result, expected)
```